src/entity/profession/ArmorerProfession.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\entity\profession;
+
+use pocketmine\block\VanillaBlocks;
+use pocketmine\data\bedrock\VillagerProfessionTypeIds;
+use pocketmine\entity\trade\TradeRecipe;
+use pocketmine\item\VanillaItems;
+use function random_int;
+
+final class ArmorerProfession extends VillagerProfession{
+
+	public function __construct(){
+		parent::__construct(VillagerProfessionTypeIds::ARMORER, "entity.villager.armor", VanillaBlocks::BLAST_FURNACE());
```

Comment:
This is a translation key, make a PR in https://github.com/pmmp/Language to later use a constant

Reply:
This is a client-side translation key, so I don't think PM-side translations are needed?

Reply:
Avoid hardcode translations, this is not the only place where translation is only used client-side, e.g. names of enchantments.